### PR TITLE
Cloning and configuring Robottelo virtualenv locally

### PR DESCRIPTION
**Pull request by rochacbruno**
As all the tasks is being tagged, the way to run when we don't want some stuff is using `--skip-tags` ans also passing --extra-vars if nor running from script.

Script now will ask a lot of things as git username, email etc..

### CODE REVIEW

**Comment by sthirugn** [comment]
ACK

File: dev/playbooks/README.md [hunk 1]
```diff
@@ -3,39 +3,81 @@ SatelliteQE Dev Environment Playbooks
...
+```bash
+$ git clone https://github.com/SatelliteQE/sat6helper
+$ cd sat6helper/dev/playbooks
+```
+
+Install dependencies
+
+```bash
+sudo dnf update -y
+sudo dnf install -y wget unzip python2 python-devel
```
Comment (elyezer) [inline dev/playbooks/README.md:53]: Is the python2 for robottelo or robottelo-ci?

**Comment by elyezer** [comment]
ACK, left a non-blocking question.

File: dev/playbooks/README.md (hunk 1 above)
Comment (rochacbruno) [inline dev/playbooks/README.md:53]: this is for ansible

**Pull request merged**